src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1323,6 +1324,13 @@ export default class DataTable extends Vue {
     if (this.data.body.length > 0) {
       const bodyRows = this._sortedData && this._sortedData.length > 0 ? this._sortedData : this._serializedDataBody;
 
+      this._printDisabledColsIndexes = [];
```

Comment:
Why emptying it again each time the body renders?

Reply:
Without initializing it first it gives me the error ""TypeError: Cannot read properties of undefined (reading 'push')"".  I can set it in created() though (?)